hc/accounts/views.py

```diff
@@ -218,6 +220,44 @@ def profile(request):
 
     return render(request, "accounts/profile.html", ctx)
 
+@login_required
+def my_reports(request):
+    # Display a report of user's current checks when
+    user = request.user
+    checks = user.check_set.order_by("created")
+
+    ctx = {
+        "page": "reports",
+        "checks": checks,
+    }
+
+    return render(request, "accounts/my_reports.html", ctx)
+
+@login_required
```

Comment:
@pndemo why do you have two similar methods? There's another `set password` method here https://github.com/andela/hc-kalii/blob/b38e100a7ac974c7d4299b550a90ba32be22f486/hc/accounts/views.py#L262-L286